### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by vicelikedust**
added notes

### CODE REVIEW

**Comment by sandrotosi** [comment]
what's `gmap3`?

**Comment by vicelikedust** [comment]
That would be my fault should just be gmap.apikey = "Enter_API_Key_Here"
I wrote my code to be gmap3

File: README.rst [hunk 1]
```diff
@@ -9,7 +9,10 @@ make creating exploratory map views effortless. Here's a crash course:
 ::
 
     from gmplot import gmplot
-
+    
+    # Add Google API Key
+    gmap.apikey = "Enter_API_Key_Here"
+    
     # Place map
     gmap = gmplot.GoogleMapPlotter(37.766956, -122.438481, 13)
```
Comment (frslm) [inline README.rst]: `gmap.apikey` is accessed before `gmap` exists? Maybe the API key can be defined as a separate variable first, and then passed into `GoogleMapPlotter`?:

```
# Add Google API Key
apikey = "Enter_API_Key_Here"

# Place map
gmap = gmplot.GoogleMapPlotter(37.766956, -122.438481, 13, apikey)
```

Comment (frslm) [inline README.rst]: `gmap.apikey` is accessed before `gmap` exists? Maybe the API key can be defined as a separate variable first, and then passed into `GoogleMapPlotter`?:

```
# Add Google API Key
apikey = "Enter_API_Key_Here"

# Place map
gmap = gmplot.GoogleMapPlotter(37.766956, -122.438481, 13, apikey)
```

**Comment by frslm** [comment]
Closing as a duplicate of #52.

**Pull request closed without merging**